README.md

```diff
@@ -95,6 +95,28 @@ We'll insert the following middlewares into the rails stack:
 2. `Rack::SslEnforcer`: enforces HTTPS.
 3. `Rack::Deflater`: compresses responses from the application, can be disabled with `ROO_ON_RAILS_RACK_DEFLATE` (default: 'YES').
 
+### Sidekiq
+
+Unless `SIDEKIQ_ENABLED` is set to false we'll check for the existance of sidekiq gems. Please ensure the following lines are included in your project Gemfile.
+
+```
+source 'https://rubygems.org' do
```

Comment:
It's just because of the sidekiq-pro gem. I don't see how we can bundle it with this gem so it needs to be added by hand.